### ISSUE TITLE
Console error each time I click the searchbar

### ISSUE THREAD

**Issue by hugoseri** [lelouch77/docusaurus-lunr-search#42]
## Expected behavior
There shouldn't have any console error when clicking the searchbar.

## Actuel behavior
When hovering the searchbar, I get the following console error:

```
main.5fef3583.js:2 Uncaught TypeError: e.handleSearchBarToggle is not a function
    at VM893 c4f5d8e4.e9d8ad2d.js:1
    at Object.l (main.5fef3583.js:2)
    at p (main.5fef3583.js:2)
    at main.5fef3583.js:2
    at b (main.5fef3583.js:2)
    at lt (main.5fef3583.js:2)
    at at (main.5fef3583.js:2)
    at ut (main.5fef3583.js:2)
    at mt (main.5fef3583.js:2)
    at F (main.5fef3583.js:2)
```

However, it doesn't affect the searchbar behavior, everything works perfectly.

## How to fix it
I simply modified `src/theme/SearchBar/index.js` at line 69 with the following:
```
      props.handleSearchBarToggle && props.handleSearchBarToggle(!props.isSearchBarExpanded);
```
I didn't go too deep to find the reason why handleSearchBarToggle wasn't defined at some point but my solution seems to fix the issue.

Maybe I should add a pull request for this ? Or maybe there is a better fix ?

**Comment by lelouch77** [comment]
Sure, Please send a pull request.

**Comment by lelouch77** [comment]
fixed in v2.1.14
